draft-ietf-quic-recovery.md

```diff
@@ -711,18 +710,18 @@ before Initial packets, early 0-RTT packets will be declared lost, but that
 is expected to be infrequent.
 
 It is expected that keys are discarded after packets encrypted with them would
-be acknowledged or declared lost.  Initial secrets however might be destroyed
```

Comment:
This "might" is a MUST.

Reply:
We could add a MUST, but there's no reason given the TLS draft has a MUST, unless we're trying to move the TLS references to informational rather than normative?